agent/util-scripts/test-bin/ssh

```diff
@@ -21,16 +21,45 @@ done
 remote=${1}
 shift
 
+function mkdirp {
+    if [[ -d ${1}/${2} ]]; then
+        return 0
+    fi
+    (cd ${1}; mkdir -p ${2})
+    return ${?}
+}
+
 if [[ "${1}" == "hostname" && "${2}" == "-s" ]]; then
     echo "${remote}"
     exit_code=0
 elif [[ "${1}" == "echo"* ]]; then
     echo "127.0.0.1 54038 ${remote} 22"
     exit_code=0
-elif [[ "$(basename -- "${1%% *}")" == "pbench-tool-meister-remote" ]]; then
+elif [[ "$(basename -- "${1%% *}")" == "pbench-tool-meister" ]]; then
     _dir=$(dirname ${0})
-    _pbench_full_hostname="${remote}" _pbench_hostname="${remote}" _tool_bin=${_dir}/mpstat ${1} localhost "${3}" "${4}" "${5}"
-    exit_code=$?
+    # To simulate running a Tool Meister instance "remotely" we reset the
+    # environment presented to the pbench-tool-meister instance. We keep the
+    # test environment variables, PATH, HOME, etc., but we also keep the
+    # pbench_tmp environment variable.  This is convenient for the test system
+    # to avoid tracking down arbitrary /var/tmp usage.
+    _remote_dir="remote/${remote}/tmp"
+    mkdirp ${_testdir} ${_remote_dir}
+    _pbench_tmp=${_testdir}/${_remote_dir}
+    env -i \
+        PATH=${PATH} \
+        HOME=${HOME} \
+        PYTHONHASHSEED=${PYTHONHASHSEED} \
+        PARALLEL_TMP=${PARALLEL_TMP} \
```

Comment:
Where does `PARALLEL_TMP` come from?  What is it used for?  And, why doesn't it have `${_remote_dir}` in it?

Reply:
From `parallel`, just passing it along.